numba/core/typing/arraydecl.py

```diff
@@ -445,9 +445,12 @@ def resolve_view(self, ary, args, kws):
 
     @bound_function("array.astype")
     def resolve_astype(self, ary, args, kws):
+        from numba.core.errors import RequireLiteralValue
```

Comment:
Please could you move this import to the top, thanks.

Reply:
Yes absolutely. Out of curiosity I thought I saw that other code segments do some imports within functions to reduce the overall import time. Is this correct and if so what is the methodology for importing inside a function.

Reply:
I understand that its clear in this example because the file already imports `numba.core.errors`.

Reply:
Thanks. In general, unless proven to be an issue, imports go at the top, it makes refactoring and dealing with side effects easier.